product/reports/670_Performance by Asset Type - Middleware Servers/110_JVM Heap Usage - daily over last week.yaml

```diff
@@ -0,0 +1,71 @@
+--- 
+where_clause: 
+dims: 
+created_on: 2016-05-23 14:53:10.822035 Z
+reserved: 
+title: "JVM Heap Usage - daily averages for last week"
+conditions: 
+updated_on: 2016-05-23 00:35:33.732482 Z
+order: Descending
+graph: 
+menu_name: "JVM Heap Usage - daily over last week"
+rpt_group: Custom
+priority: 
+col_order: 
+- id
+- name
+- start_time
+- mw_heap_used_min
+- mw_heap_used_avg
+- mw_heap_used_max
+- mw_heap_committed_min
+- mw_heap_committed_avg
+- mw_heap_committed_max
+- mw_heap_max_min
+- mw_heap_max_avg
+- mw_heap_max_max
+timeline: 
+id: 99
+file_mtime: 
+categories: 
+rpt_type: Custom
+filename: 
+db_options:
+  :rpt_type: middleware
+  :options:
+    :interval: daily
+    :start_offset: 604800
+    :end_offset: 0
+include: {}
+
+db: MiddlewareServerPerformance
+cols: 
+- id
+- name
+- start_time
+- mw_heap_used_min
+- mw_heap_used_avg
+- mw_heap_used_max
+- mw_heap_committed_min
+- mw_heap_committed_avg
+- mw_heap_committed_max
+- mw_heap_max_min
+- mw_heap_max_avg
+- mw_heap_max_max
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1662329/16519026/88ae7c44-3f86-11e6-8e4f-bd33a7f76f42.png)
These reports are pretty preliminar and they are not driven yet.
I introduced it as it is a metric that it has a variation, and from statistically point of view in a report like the attached it might be interesting to check when heaps reached max and what was that value.
The goal of this PR as similar like the alerting ones are focus on the architecture to add support for reporting and alerting.
Once we have a solid integration between miq and hawkular we can refine and adjust these reports.